test/libsolidity/semanticTests/array/push/push_no_args_bytes.sol

```diff
@@ -23,7 +23,7 @@ contract C {
 // ----
 // l() -> 0
 // g(uint256): 70 ->
-// gas irOptimized: 429679
+// gas irOptimized: 434229
```

Comment:
Do you know why this increased so much?

Reply:
It is in a loop, so it does 70 additional checks (because there was no check before, and that was a bug)

Reply:
We really need to get those out of the loop somehow at some point...

Reply:
> We really need to get those out of the loop somehow at some point...

I am willing to check and do that work, just after I merge outstanding PRs